### PR TITLE
Add functionality to fetch dividends

### PR DESCRIPTION
**Pull request by vsimon**

### CODE REVIEW

File: tests/resources/test_dividend.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from fast_arrow import util
```
Comment (westonplatter) [inline tests/resources/test_dividend.py]: do you need this import?

Comment (westonplatter) [reply tests/resources/test_dividend.py]: I'll merge and probably take this out. Thanks for the PR.

**Comment by westonplatter** [comment]
@vsimon thanks for the PR.

File: tests/resources/test_dividend.py (hunk 1 above)
Comment (vsimon) [reply tests/resources/test_dividend.py]: Thanks. The other tests had this and you're right its not needed. Yes, it can be cleaned up in a follow-up PR.

**Pull request merged**